spec/requests/file_upload_spec.rb

```diff
@@ -171,19 +182,20 @@
       end
 
       before do
+        s3.stub_responses(
+          :head_object,
+          'NotFound',
+          Aws::S3::Types::HeadObjectOutput.new(last_modified: Time.now.utc)
+        )
+        s3.stub_responses(:put_object, {})
+
         post '/service/service-slug/user/user-id', params: json.to_json, headers: headers
       end
 
       it 'has status 201' do
         expect(response).to have_http_status(201)
       end
 
-      it 'saves the decoded data to a local file' do
```

Comment:
Remove this spec as it was testing functionality that only existed in test/dev and not in production.